canonical-websites/build.snapcraft.io
# Updates header to with new snapcraft navigation

## Done

Updates header and navigation to be consistent with snapcraft.io.

Fixes #1047 

## QA

- Check out this feature branch
- Run the site using the command ```npm start -- --env=environments/dev.env```
- View the site locally in your web browser at: [http://0.0.0.0:8000/](http://0.0.0.0:8000/)
- Navigation should have Vanilla styles and items same as on snapcraft.io
- Sign in
- Navigation should show that you are signed in




## Screenshots

Not signed in

<img width="1059" alt="screen shot 2018-01-19 at 16 36 43" src="https://user-images.githubusercontent.com/83575/35158567-bf5b4720-fd37-11e7-9c27-2890ae739f39.png">

Signed in

<img width="1068" alt="screen shot 2018-01-19 at 16 36 55" src="https://user-images.githubusercontent.com/83575/35158572-c4517c0e-fd37-11e7-93d8-bfa45f6ff553.png">
